src/mame/drivers/badlands_ms.cpp

```diff
@@ -184,6 +185,9 @@ ROM_START( badlandsm )
 	ROM_REGION( 0x120, "proms", 0 )    // PROMs (function unknown)
 	ROM_LOAD( "mod-1-4_sound_105_82s123.ic20", 0x000, 0x020, CRC(14d72781) SHA1(372dc021d8aaf4aa6fd46e69a3d8f1c68113426f) )
 	ROM_LOAD( "mod-51-1_502_82s129a.ic10",     0x020, 0x100, CRC(15085e44) SHA1(646e7100fcb112594023cf02be036bd3d42cc13c) )
+	ROM_LOAD( "mod_21_1_205_82s129.ic12",      0x120, 0x100, CRC(204a7aee) SHA1(322164134aa65c37a9389024f921364a81d13e88) )
+	ROM_LOAD( "mod_21_1_210_82s129.ic4",       0x220, 0x100, CRC(d3e676aa) SHA1(241a9f9df88e8f796ca8c773ae7930f6e7ce4db8) )
+
 
 	ROM_REGION( 0x1000, "plds", ROMREGION_ERASEFF )
```

Comment:
Fixed, thanks

Reply:
> Fixed, thanks

This response doesn’t instill much confidence that you’ll do the right thing going forward.  Is it clear that it’s your responsibility to build, validate and test your changes *before* opening a pull request, rather than wasting other people’s time catching obvious errors?

Reply:
Yes!